include/vinecopulib/vinecop/implementation/tools_select.ipp

```diff
@@ -97,9 +97,20 @@ inline VinecopSelector::VinecopSelector(const Eigen::MatrixXd& data,
   , controls_(controls)
   , pool_(controls_.get_num_threads())
   , trees_(std::vector<VineTree>(1))
+  , threshold_(controls.get_threshold())
+  , psi0_(controls.get_psi0())
 {
-  threshold_ = controls.get_threshold();
-  psi0_ = controls.get_psi0();
+  vine_struct_ = RVineStructure(tools_stl::seq_int(1, d_), 1, false);
+}
+
+inline VinecopSelector::VinecopSelector(const Eigen::MatrixXd& data,
+                                        const RVineStructure& vine_struct,
+                                        const FitControlsVinecop& controls,
+                                        std::vector<std::string> var_types)
+  : VinecopSelector(data, controls, var_types)
+{
+  vine_struct_ = vine_struct;
+  proximity_ = false;
```

Comment:
I wonder whether there's a better name than `proximity`. Maybe something like `structure_provided_` or `tree_fixed`?

Reply:
Meh, you really like those better? I don't really care...

Reply:
It's not that I like the others so much, but `proximity_` is at best obscure:
- The proximity condition needs to be satisfied no matter if `proximity_` is true or false.
- When `proximity_ == false`, the proximity condition is always true and we don't have to check for it.
- The proximity condition is really not the important aspect in this whole thing.

So I think we should find a name that better conveys the meaning of this flag. When reading the `if` switches, I think we mean
```
if (the structure in this tree is unknown) {
  // loop through all possible edges
} else { 
  // use the known edges
}
```
so the flag's name should optimally convey this. Other suggestions: `structure_known`, `tree_structure_known`, `select_edges`, ... I don't really care about the specific name as long as it can be understood without understanding the code inside the if switches.

Reply:
I like `structure_known`, I push with this name..